pyproject.toml

```diff
@@ -293,6 +293,10 @@ skip_covered = false
 
 [tool.coverage.paths]
 source = ["src", "**/site-packages"]
+attrs-plugin = [
+  "tests/_trio_check_attrs_aliases.py",
+  "_trio_check_attrs_aliases.py",
```

Comment:
Huh, why this? I guess this is necessary to merge the importable with the file under `tests/`?

Reply:
FWIW, looking at one of the codecov uploads (https://storage.googleapis.com/codecov-production/shelter/github/python-trio%3A%3A%3A%3Atrio/ffe7261e66a3f7af53e9671c9fc06e2342d3bda6/de0c6baf-a91a-4026-bc82-3382eb60fda8.txt?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=GOOG1EJOGFN2JQ4OCTGA2MU5AEIT7OT5Z7HTFOAN2SPG4NWSN2UJYOY5U6LZQ%2F20251206%2Fus-west2%2Fs3%2Faws4_request&X-Amz-Date=20251206T014925Z&X-Amz-Expires=30&X-Amz-SignedHeaders=host&X-Amz-Signature=4623e30cb5fbffcc5f205f54c40f2a8bb795d3b626d339e6763848ec2038497f), it contains `<class name="_trio_check_attrs_aliases.py" filename="tests/_trio_check_attrs_aliases.py" complexity="0" line-rate="1" branch-rate="1">` so Codecov doesn't have a problem mapping it back to the repo.